augur/utils.py

```diff
@@ -84,16 +84,26 @@ def is_date_ambiguous(date, ambiguous_by="all"):
     ambiguous_by : str
         Field of the date string to test for ambiguity ("day", "month", "year", "all")
     """
-    year, month, day = date.split('-')
+    date_components = date.split('-', 2)
+
+    if len(date_components) == 3:
+        year, month, day = date_components
+    elif len(date_components) == 2:
+        year, month = date_components
+        day = "XX"
+    else:
+        year = date_components[0]
+        month = "XX"
+        day = "XX"
+
     return (
```

Comment:
Is the indentation of the conditionals like this for a reason? I would have expected similar to how the above if/else block is indented (and my linter agrees!)

Reply:
I missed this in my original code review of #623! :) Thank you for catching it.